apps/mail-bridge/postal-db/functions.ts

```diff
@@ -53,6 +55,51 @@ export async function createOrg(input: CreateOrgInput) {
   };
 }
 
+export async function removeOrg(orgPublicId: TypeId<'org'>) {
```

Comment:
we should pass in orgId not orgPublicId since this is an internal procedure

Reply:
postal doesn't know about the org id, it can identify using publicId only, we cant do a query as the org has already been deleted before this query

Reply:
ah fair enough